hack/etcdrecovery.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Can't you reuse upgrade.sh?

Reply:
I'm fine with the separate script, but it should be minimal and calling a separate helper prior to rolling it in after the admin API work